### PR TITLE
Add warehouse tests

### PR DESCRIPTION
**Pull request by xonism**

### CODE REVIEW

File: internship-warehouse/src/test/java/com/twoday/internshipwarehouse/services/ProductServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ProductServiceTests {
+
+    @Mock
+    private ProductRepository productRepository;
+
+    @InjectMocks
+    private ProductService underTest;
```
Comment (ArnasPetraitis) [inline internship-warehouse/src/test/java/com/twoday/internshipwarehouse/services/ProductServiceTests.java]: I would leave normal naming, even if it has mocked dependencies. `ProductService productService;` but mocked services may be named `*Mock`

File: internship-shop/src/test/java/com/twoday/internshipshop/TestHelpersTests.java [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.twoday.internshipmodel.OrderCreateRequest;
+import com.twoday.internshipshop.utils.JsonUtils;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
-class JsonUtilsTests {
+class TestHelpersTests {
```
Comment (ArnasPetraitis) [inline internship-shop/src/test/java/com/twoday/internshipshop/TestHelpersTests.java]: Personaly, I'm used to `*Test` naming. Do these tests run on maven build now?

**Review by ArnasPetraitis — CHANGES_REQUESTED** [review]
Minor changes, tests are good

File: internship-warehouse/src/test/java/com/twoday/internshipwarehouse/services/ProductServiceTests.java (hunk 1 above)
Comment (xonism) [reply internship-warehouse/src/test/java/com/twoday/internshipwarehouse/services/ProductServiceTests.java]: I think the normal naming everywhere would be best, to not clutter it too much 😄

File: internship-shop/src/test/java/com/twoday/internshipshop/TestHelpersTests.java (hunk 2 above)
Comment (xonism) [reply internship-shop/src/test/java/com/twoday/internshipshop/TestHelpersTests.java]: I'll change the naming but they did run, these patterns are recognized by Maven:

image

**Review by ArnasPetraitis — APPROVED** [review]

**Pull request merged**